CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## `v14.1.1`
+
+- Fixing timestamp marshalling bug in the `storage` package.
+- Updating `profileBuilder` to clear-output folders when it is run by `go generate`.
+- Tweaking Swagger -> SDK config to use "latest" instead of "nightly" and be tied to a particular version of autorest.go.
```

Comment:
Nit: from what I see it is tied to particular minor version of autorest.go. Any reason we don't want to tie it to the major version ? That would still mean no breaking changes

Reply:
It is tied to the major version, see below. :)

Reply:
I was referring that this yes, to say "particular major version" :D